ports/raspberrypi/boards/cytron_maker_pi_rp2040/board.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Scott Shawcroft for Adafruit Industries
```

Comment:
Please update the copyright information.  It is appropriate to include your name as well as the name of the company for whom the work was done, if applicable.

Reply:
I'm sorry I missed that out.
It's already updated.
Thanks.